src/main/resources/edu/cornell/kfs/sys/cu-spring-sys.xml

```diff
@@ -264,4 +264,82 @@
 	<bean id="paymentSourceHelperService" parent="paymentSourceHelpService-parentBean"
 			class="edu.cornell.kfs.sys.document.service.impl.CuPaymentSourceHelperServiceImpl"/>
 
+	<bean id="modulesLockJob" parent="modulesLockJob-parentBean">
+		<property name="steps">
+			<list>
+				<ref bean="lockARModuleStep"/>
```

Comment:
The problem with the current steps from KFS is that they include one for locking CAB, even though we no longer have that module or a parameter for locking it. This will be fixed in base code once we move to the 2017-09-07 KualiCo patch, but if we want this feature to move forward, we need some other workaround.

KFS does come with some field-overriding helper classes/beans, though. Perhaps we can use one of those to remove the problematic element, and then we can remove such a helper bean once we migrate to the 2017-09-07 patch. I'll see if that will work, so that we don't have to re-insert the CAB parameter or do some other workaround like that.